docs/footer.ftd

```diff
@@ -0,0 +1,89 @@
+-- import: fifthtry.github.io/admint.io/assets
+-- import: fifthtry.github.io/doc-site as ds
+-- import: fifthtry.github.io/admint.io/docs
+-- import: fifthtry.github.io/admint.io/lib
+-- import: fifthtry.github.io/admint-cs as cs
+-- import: fifthtry.github.io/admint.io/common
+-- import: fifthtry.github.io/admint-typography as typo
+
+
+
+
+
+-- ds.page: Footer
+show-footer: true
+site-logo: $assets.files.static.admint-logo.svg
+colors: $cs.main
+types: $typo.types
+$widthfill: true
+sections: $lib.sitemap.sections
+
+
+-- ds.page.footer:
+
+-- lib.footer:
+site-logo: $assets.files.static.footer.admint-logo-footer.svg
+site-url: https://admint.io/
+footer-links: $common.footer-links
+social-links: $common.social-links
+team-thumb: $assets.files.static.footer.footer-hand-thumb.svg
+
+-- end: ds.page.footer
+
+-- ds.page.body:
+
+We have developed the `footer` component for [admint.io](https://admint.io/), 
+which matches the design currently displayed on the live site.
+
+-- ds.h3: How to use?
+
+To use the `footer` component you need to import `lib.ftd` & `common`  inside
+your `.ftd` file. The `footer` component uses `footer-links` which are added
```

Comment:
Add social-links as well which is required for this footer.
The `footer` component uses `footer-links` and `social-link` which are defined
inside `common/index.ftd` file.